### ISSUE TITLE
Please add Sunny Bounder

### ISSUE THREAD

**Issue by Xappers** [punkUser/xwing_math#17]
I would be very interested to know the statistical implication of Sunny Bounder's pilot ability.
'While you defend or perform an attack, if you have the same result on each of your dice, you may add 1 matching result.'

**Comment by punkUser** [comment]
Yeah Sunny is interesting but he has a lot of weird interactions with other effects that haven't been clarified in the FAQ to date. Some of them probably will never be clarified because they can't currently be used together in the game (different faction cards, etc) but they still need to have some sort of reasonable implementation in the calculator.

Certainly on the TODO list but due to the complexity : benefit ratio it's a bit lower down the list.
